include/SW_datastructs.h

```diff
@@ -689,6 +739,22 @@ typedef struct {
 
 } SW_SOILWAT;
 
+typedef struct {
+	FILE *logfp; /* REQUIRED; defined in SW_Main_lib.c */
```

Comment:
The comment `/* REQUIRED; defined in SW_Main_lib.c */` doesn't apply anymore for both `*logfp` and `logged`